examples/helloworld_shm/HelloWorldDataShm.idl

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright(c) 2006 to 2020 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+module HelloWorldDataShm
+{
+  struct Msg
+  {
+    long data;
```

Comment:
It is enough to just have this small of a message?

Maybe it is better to have a message with a huge payload to better illustrate the performance increase iceoryx can cause?